File: diary/src/main/kotlin/diary/pojo/vo/DailyLogVO.kt
package diary.pojo.vo

import java.time.LocalDate

// 日期分组的日志视图对象
data class DailyLogVO(
    var id  : Int,
    var date: LocalDate, // 日期
    var weather: String, // 天气情况
    var logs: MutableList<LogEntryVO>, // 该日期下的所有日志条目
)

File: Vue3/src/diary/types/vo/DailyLogVO.ts
import type {LogEntryVO} from './LogEntryVO'
import type {Weather} from "@/diary/types/vo/Weather";

export interface DailyLogVO {
    id: number;
    date: string; // 前端使用字符串格式更方便
    weather: Weather;
    logs: LogEntryVO[];
}
